### PR TITLE
Rework tests

### PR DESCRIPTION
**Pull request by TAG-Epic**

### CODE REVIEW

**Comment by TAG-Epic** [comment]
No update in 3 days... doing a pr to the main repo

**Comment by devoxin** [comment]
Why add a bunch of whitespace and ignore it in flake8? Seems pretty useless to me, and while this may not be seen as a valid concern -- also bumps up the disk space usage of each file.

File: ksoftapi/__init__.py [hunk 1]
```diff
@@ -13,9 +13,9 @@
 import logging
 from collections import namedtuple
 
-from .client import Client
-from .errors import APIError, Forbidden, NoResults
-from .events import BanUpdateEvent
+from .client import Client  # noqa
+from .errors import APIError, Forbidden, NoResults  # noqa
+from .events import BanUpdateEvent  # noqa
```
Comment (devoxin) [inline ksoftapi/__init__.py:18]: Why broad ignore?

File: .travis.yml [hunk 2]
```diff
@@ -14,4 +14,3 @@ install:
 
 script:
   - flake8
-  - pylint ksoftapi
```
Comment (devoxin) [inline .travis.yml]: Why? Pylint can pick up things flake8 doesn't and vice-versa. flake8 seems more useful for "picking up "garbage" (e.g. extraneous whitespace) while pylint seems to cover some garbage detection and also code smells (invalid syntax/cyclic imports etc).

File: ksoftapi/apis/bans.py [hunk 3]
```diff
@@ -15,19 +15,19 @@ def __init__(self, client):
         self._client = client
         self._hooks = []
         self._last_update = time() - 60 * 10
-
+
```
Comment (AndyTempel) [inline ksoftapi/apis/bans.py:18]: Unnecessary whitespace should not be present in the files.

File: ksoftapi/__init__.py (hunk 1 above)
Comment (AndyTempel) [reply ksoftapi/__init__.py:18]: ^

File: ksoftapi/http.py [hunk 4]
```diff
@@ -18,59 +18,59 @@
 
 class HttpClient:
     BASE = 'https://api.ksoft.si'
-
+    
     def __init__(self, authorization, loop):
         self._default_headers = {
             'Authorization': 'NANI ' + authorization,
```
Comment (AndyTempel) [inline ksoftapi/http.py:24]: Forgot about this. Change NANI to Bearer for future compatibility.

File: run_tests.py [hunk 5]
```diff
@@ -1,45 +0,0 @@
...
-from io import StringIO
-
-import pylint.lint as pylint
-from flake8.api import legacy
-from pylint.reporters import text
-
-
-def test_flake8():
-    style_guide = legacy.get_style_guide()
-    report = style_guide.check_files(['lavalink'])
```
Comment (AndyTempel) [inline run_tests.py:11]: Lavalink *hmmmmmmm*

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = .git, __pycache__, build, dist
```
Comment (AndyTempel) [inline setup.cfg:2]: Perhaps venv and/or env should be added too.

File: .travis.yml [hunk 7]
```diff
@@ -1,14 +1,15 @@
...
 
-
+cache: pip
 install:
   - pip install flake8 pylint
 
 script:
-  - python run_tests.py
\ No newline at end of file
+  - flake8
```
Comment (AndyTempel) [inline .travis.yml:15]: Think about including CI

**Review by AndyTempel — CHANGES_REQUESTED** [review]

**Comment by devoxin** [comment]
Sorry. I don't really agree with this PR as a whole. The point of tests is to ensure the library meets a certain level of quality (or, standard if you will). By removing pylint from the tests, ignoring codes that have no reason to be ignored and inserting "garbage" into the code just seems counterproductive. It is for these reasons that I have decided to close this pull request.
